### PR TITLE
nav bar with drop down shadow on scroll

### PR DESCRIPTION
**Pull request by Anupam-dagar**
live link:
http://anupamdagar.me/hack-in-the-north-preview-repo/main

### CODE REVIEW

**Comment by gauthamzz** [comment]
this is great ,is this done ?

**Comment by Anupam-dagar** [comment]
is transparency required in navbar like it is there in cloudcv evalai?

**Comment by gauthamzz** [comment]
I think this works fine

**Comment by Anupam-dagar** [comment]
yes, PR can be merged.

File: main/index.html [hunk 1]
```diff
@@ -19,14 +19,15 @@
         <link rel="stylesheet" href="static/css/main.css">
         <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css">
         <link href="static/css/hover-min.css" rel="stylesheet" media="all">
+        <link rel="stylesheet" type="text/css" href="static/css/styling.css">
```
Comment (gauthamzz) [inline main/index.html]: this is a very ambiguous file name . Rather make it more specific to the function and add comments

File: main/index.html [hunk 2]
```diff
@@ -233,6 +213,16 @@ <h1 class="contact-title">CONTACT</h1>
         </script>
         <script src="https://www.google-analytics.com/analytics.js" async defer></script>
         <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/wow/1.1.2/wow.min.js"></script>
+        <script type="text/javascript">
```
Comment (gauthamzz) [inline main/index.html]: lint this ?

Comment (Anupam-dagar) [reply main/index.html]: tried on jshint to check for warnings, it returned 0 warnings.

File: main/index.html (hunk 1 above)
Comment (Anupam-dagar) [reply main/index.html]: changed to cards.css

**Pull request merged**